dev/template.env

```diff
@@ -37,8 +37,8 @@ NB_FAPI_PORT_HOST=8080
 NB_FAPI_TAG=latest
 
 # ---- CONFIGURATION FOR QUERY TOOL ----
-# URL of the f-API as it will appear to a user
-NB_API_QUERY_URL=http://localhost:8080
+# URL of the n-API/f-API the query to will talk to, as it appears to a user
+NB_API_QUERY_URL=http://XX.XX.XX.XX  # REPLACE WITH YOUR API URL (AND PORT, IF APPLICABLE) AS IT APPEARS ON A USER'S MACHINE, OR localhost:<port> IF QUERY TOOL WILL ONLY BE ACCESSED FROM ITS HOST MACHINE
```

Comment:
I'd say: let's default to having comments above the line. We already saw some cases where inline comments break - and they also look pretty bad if they are this long.